.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,7 @@
+settings:
+  jira_project_key: MULTI
+  labels:
+    - jira
+  status_mapping:
+    opened: Untriaged
+    closed: Done
```

Comment:
Perhaps the label mapping from the example in the readme would make sense too?

```
  label_mapping:
    enhancement: Story
```

Reply:
see my global comment.